src/manifest.ts

```diff
@@ -115,6 +115,7 @@ export interface ManifestOptions {
   releaseLabels?: string[];
   draft?: boolean;
   draftPullRequest?: boolean;
+  groupPullRequestTitlePattern?: string;
```

Comment:
I decided against naming it `mergedPullRequestTitlePattern` due to the keyword `merge` being ambiguous in this context

Reply:
Open to suggestions, e.g. `pull-request-group-title-pattern`, etc.